material/src/main/java/org/odk/collect/material/MaterialProgressDialogFragment.java

```diff
@@ -52,22 +59,35 @@ public String getMessage() {
         return getArguments().getString(MESSAGE);
     }
 
+    public @DrawableRes int getIcon() {
```

Comment:
Those icons do not look good I've been thinking about removing them for a while. @seadowg what do you think it looks like we display them in only two dialogs so maybe we should be consistent and remove it at all?

Reply:
What icons are you referring to? It won't let me navigate to the commented code as the changeset is out of date.

Reply:
This Icon, I think it is used only in two dialogs:
![Screenshot_1644256568](https://user-images.githubusercontent.com/3276264/152844877-196d40d8-9b25-4965-9462-b00a26dda510.png)

Reply:
This is the (i) icon, right? I would really appreciate it going away. That was the intent as dialogs got reworked. It was a pattern in Android a really long time ago.

Reply:
Exactly it's something we don't add to our new dialogs so it exists in 2-3 old ones.